src/hooks/useActiveProject.tsx

```diff
@@ -31,6 +32,9 @@ const findProjectAndSubjectById = (
     const defaultSubject = subjects?.find(
       (s) => s.projectId === defaultProject?.id,
     );
+    defaultSubject?.subjectId &&
+      _sdkTracker.identifyUser(defaultSubject?.subjectId);
+    _sdkTracker.userPropertyUpdate('project', defaultProject?.id);
```

Comment:
Why is `project` considered a "user property"?

Reply:
Also, should we only be updating the property when we know there's a project? It seems like in this case we could go from setting it to `undefined` first before setting it back to an actual value.